CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.39.111](https://github.com/brave/brave-browser/releases/tag/v1.39.111)
+
+ - Added Brave VPN. ([#12197](https://github.com/brave/brave-browser/issues/12197))
```

Comment:
Shouldn't this be `Brave Firewall + VPN` ? Ref: `https://brave.com/firewall-vpn/`